core/server/config/overrides.json

```diff
@@ -64,5 +64,18 @@
     },
     "maintenance": {
         "enabled": false
+    },
+    "api": {
+        "versions": {
+            "active": {
+                "admin": "v2/admin",
+                "client": "v2/client"
```

Comment:
`/client`? 🤔

Reply:
Haha! 🤐 Not sure what I was thinking about `client` while editing this, for some reason that just stuck out of brain even here! Updating this silly 🐛